client/configuration_test.go

```diff
@@ -451,3 +464,42 @@ func TestConnectionStringWithCertificateFile(t *testing.T) {
 	require.NotNil(t, config)
 	assert.NotNil(t, config.RootCAs)
 }
+func TestConnectionStringWithKeepAlive(t *testing.T) {
+	// KeepAliveInterval
+	config, err := client.ParseConnectionString("esdb://user:pass@127.0.0.1/?keepAliveInterval=zero")
```

Comment:
Could we add also tests with both `keepAliveInterval` and `keepAliveTimeout` provided all is working fine (e.g. `keepAliveInterval` equal to `-1` and `keepAliveTimeout` provided etc.).